gadget-container/gadgettracermanager/main.go

```diff
@@ -118,8 +118,12 @@ func main() {
 		defer conn.Close()
 		client = pb.NewGadgetTracerManagerClient(conn)
 
-		ctx, cancel = context.WithTimeout(context.Background(), clientTimeout)
-		defer cancel()
+		if liveness {
+			ctx = context.Background()
+		} else {
+			ctx, cancel = context.WithTimeout(context.Background(), clientTimeout)
+			defer cancel()
+		}
```

Comment:
I'm a little bit worried about k8s < 1.20 as according to the [docs](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/.): "Before Kubernetes 1.20, the field timeoutSeconds was not respected for exec probes: probes continued running indefinitely, even past their configured deadline, until a result was returned.", so I think in that case the probe won't work as it'll never return, right? 

What about defining a big timeout like 1minute just to be sure it works on an old cluster too?

Reply:
> What about defining a big timeout like 1minute just to be sure it works on an old cluster too?
>
Mmmm good question. Yeah, maybe keeping an internal **big** timeout could help in those systems.

Reply:
Done